samples/apps/copilot-chat-app/scripts/Start-Frontend.ps1

```diff
@@ -0,0 +1,39 @@
+<#
+.SYNOPSIS
+Initializes and runs the Copilot Chat frontend.
+
+.PARAMETER ClientId
+The client (application) ID for your AAD app registration.
+
+.PARAMETER TenantId
+The tenant (directory) ID for your AAD app registration.
+If you are using a personal MSA account, enter 'msa' here.
+Defaults to the 'common' endpoint.
+#>
+
+#Requires -Version 6
+
+param (
+    [Parameter(Mandatory)]
+    [string] $ClientId,
+
+    [string] $TenantId = 'common'
+)
+
+cd "$PSScriptRoot/../WebApp"
+$EnvFilePath = "$PSScriptRoot/../WebApp/.env"
+
+# Overwrite existing .env file
+Set-Content -Path $EnvFilePath -Value "REACT_APP_BACKEND_URI=https://localhost:40443/"
```

Comment:
This will only work with local deployments. In the future, we could have https://localhost:40433 be the default for a third, optional parameter.  Not blocking for now.

Reply:
Let's keep these scripts focused on the localhost scenario.